autoconnect/autoconnect-ws/autoconnect-ws-sm/src/unidentified.rs

```diff
@@ -280,12 +294,9 @@ mod tests {
             db: hello_db().into_boxed_arc(),
             ..Default::default()
         });
-        // Ensure that we do not need to pass the "use_webpush" flag.
-        // (yes, this could just be passing the string, but I want to be
-        // very explicit here.)
-        let json = serde_json::json!({"messageType":"hello"});
-        let raw = json.to_string();
-        let msg = ClientMessage::from_str(&raw).unwrap();
+        // Using hello_json helper ensures consistent message type strings
```

Comment:
It also breaks the test.
We want to ensure that `use_webpush` is _*NOT*_ required by the `hello` message, even though it is commonly passed by clients. Thus why we were hand building things.